modules/ipc/src/zenoh/scout.cpp

```diff
@@ -103,8 +103,7 @@ auto getListOfNodes() -> std::vector<NodeInfo> {
   ScoutDataManager manager;
 
   auto config = ::zenoh::Config::create_default();
-  ::zenoh::scout(
-      std::move(config), [&manager](const auto& hello) { manager.onDiscovery(hello); }, []() {});
+  ::zenoh::scout(std::move(config), [&manager](const auto& hello) { manager.onDiscovery(hello); }, []() {});
```

Comment:
seems liks sth with the max line length changed? this seems to be a veery long line, no?

Reply:
we have a limit of 110 and that line is 109 😅